bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```diff
@@ -332,6 +332,11 @@ data:
         ipsec_encapsulation=true
       fi
 {{ end }}
+
+{{ if eq .OVNIPsecEncap "Always" }}
```

Comment:
n00b question but this Key could be unset so what happens in that scenario - did you check?

Reply:
It's expected that CNO reconcile loop renders new config according to API changes, so this must work by rolling out ovnkube-node daemonset for changes.
@huiran0826 has done some premerge test on this PR before. @huiran0826 have you also covered this case ?